src/main/java/com/microsoft/sqlserver/jdbc/SQLServerClob.java

```diff
@@ -207,6 +230,7 @@ public InputStream getAsciiStream() throws SQLException {
     public Reader getCharacterStream() throws SQLException {
         checkClosed();
 
+        getBytesFromStream();
         Reader getterStream = new StringReader(value);
         activeStreams.add(getterStream);
```

Comment:
I'm not sure this is needed because the underlying stream is already registered in `activeStreams`.